src/plugins/console/public/lib/row_parser.ts

```diff
@@ -41,18 +41,6 @@ export default class RowParser {
     }
     let line = (this.editor.getLineValue(lineNumber) || '').trim();
 
-    // Check if the line has variables, depending on the request type, (e.g. single line, multi doc requests) return the correct mode
```

Comment:
I couldn't understand why this code was needed but this code was the main factor to divide a bulk request into 2.

Reply:
I do see this logic was added as part of the initial implementation - https://github.com/elastic/kibana/pull/134215/files#diff-2caf7570c38696681df487677511180b9b56e89127b16df2e9aaf0b5db436cf8R44

I'd like to do some more testing to make sure removing it doesn't cause any regressions.